cpp/fit_dimuon.C

```diff
@@ -41,11 +41,14 @@
 using namespace std;
 using namespace RooFit;
 
-bool drawFits = true;
-bool drawResidual = false;
+bool doBinnedFit = false;
+bool categorizeSignal = false;
+bool categorizeBackground = true;
```

Comment:
I chose to stay flexible for the time being, mostly in case we see that we need to allow for different shapes for the signal across different categories.
For background, it's natural to choose different shapes per category, but I preferred flexibility in case combine doesn't like this (logical) option.